api/http/handler/fdo/handler.go

```diff
@@ -0,0 +1,30 @@
+package fdo
```

Comment:
didn't notice before, but this should go inside the `api/http/handler/hostmanagement` together with the `openamt` package